nick89mck/product-preview-card
# Readme

## Updates README to reflect final project. 
---
### Test
---
1. Make sure screenshot image is being displayed (may need to be resized)
2. Test links and fix broken ones.
3. Check spelling and grammar.
